8/README.MD

```diff
@@ -78,26 +70,26 @@ printf("%d, %c\n", a, c);
 In Python:
 ```
 a = 7
-c = 'x'
+c = "x"
 
 # Method 1
-print(a, end = '')
-print(', ', end = '')
+print(a, end = "")
+print(", ", end = "")
 print(c)
 
 # Method 2
 # Here we use the + operator to concatenate strings and the str function to convert other variables to strings.
 print(str(a) + ", " + str(c))
 
 # Method 3
-print("{0}, {1}".format(a, c))
+print("{}, {}".format(a, c))
```

Comment:
No need to hardcode indexes unless you really need to flip things (and even then, probably best to reorder arguments to `format` if possible.